docs/screens/authenticating.md

```diff
@@ -14,14 +14,14 @@
 | When application gets access token and authentication finished with success. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "authentication"<br />   "success": true<br />   "screen_name": "authenticating_screen"<br />}<pre/> |
 |:----------------------------------------------------------------------------:|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 
-| When application doesn't get access token and  When authentication finished with failure. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "authentication"<br />   "success": false<br />   "screen_name": "authenticating_screen"<br />}<pre/> |
-|:-----------------------------------------------------------------------------------------:|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| When application doesn't get access token and  When authentication finished with failure. | <pre>{<br />   "name": "action_finished"<br />   "item_name": "authentication"<br />   "success": false<br />   "screen_name": "authenticating_screen"<br />   "error_message": errorMessage<br />}<pre/> |
```

Comment:
Can you give me an advice, how I should fill value of the `error_message`?

Reply:
"Some error message", or "Client request(POST https://api.github.com/repos/exampleOwner/exampleRepo/issues/1/comments) invalid: 404 Client Error. Text: \"\"" if you prefer :)